### PR TITLE
KIALI-2085 update multi match checker validation message for DR

### PR DESCRIPTION
**Pull request by xeviknal**
Making DR multimatch validation consistent with the same one for Gateways.
https://github.com/kiali/kiali/pull/712

### CODE REVIEW

File: content/documentation/overview/index.adoc [hunk 1]
```diff
@@ -125,9 +125,9 @@ This section lists all the validations that Kiali performs on all Istio configur
 .List of destination rule validations
 |===
 |Validation message |Severity |Description |Source |Example
-|More than one Destination Rules for same host subset combination
+|More than one Destination Rules for the same host subset combination
 |warning
-|Warning shown when 2 Destination Rules points to the same host and shares one or more subsets
+|Warning shown when 2 Destination Rules point to the same host and shares one or more subsets
```
Comment (abonas) [inline content/documentation/overview/index.adoc]: shares-> share

**Comment by xeviknal** [comment]
@aljesusg @abonas changes approached :upside_down_face:

**Comment by xeviknal** [comment]
any update in this one?

**Comment by aljesusg** [comment]
@abonas can we merge this?

**Pull request merged**